x/multisig/tx.go

```diff
@@ -0,0 +1,19 @@
+package multisig
+
+import (
+	"github.com/iov-one/weave"
+)
+
+// MultiSigTx is an optional interface for a Tx that allows
+// it to support multisig contract
+type MultiSigTx interface {
+	GetMultiSig() weave.Address
+}
+
+func MultiSigConditions(contract Contract) []weave.Condition {
+	var conditions []weave.Condition
+	for _, sig := range contract.Sigs {
```

Comment:
Are these `Sigs` `Conditions` or `Addresses`?

I think there is a valid point each way (Conditions are more transparent, Addresses are easier to get and read from the blockchain). Generally, we just use Conditions in the db when we need to be clear what controls it (eg. Escrow controlled by private key or by hash preimage... big difference)

My idea is to use `Addresses` here for simplicity, but when you construct the app, you can limit the Authenticator accepted by `MultiSig` to only accept signatures not preimages. Also, when you form it, you generally need some out of bounds check to ensure you trust all the parties, that these addresses are really controlled by the proper people, etc.

If you have a good counter-argument for the use of Conditions, I am happy to hear and consider it.

Reply:
I intended to use `Addresses` indeed. However I wanted to use the `HasNConditions` function from the context to activate the contract hence the conversion to `Conditions`.
Are you saying I should move this function elsewhere / rename it to avoid any confusion about whats intended to be used with the contract (eg. `Addresses`) ?

Reply:
There should be a HasNAddresses() function as well... if not, then please add that to the `x/auth.go` helpers